hypertrace-trace-enricher/helm/values.yaml

```diff
@@ -91,6 +91,7 @@ traceEnricherConfig:
   # All other streams config goes here.
   # Remove the flower braces and add key: value pair here.
   extraKafkaStreamsConfig: {}
+  enricherExecutorsThreadPoolSize: 4 # should be atleast set equal to kafkaStreamsConfig.numStreamThreads
```

Comment:
Comment this value out. Otherwise this becomes the default. We just need a way to override it if needed.

Reply:
The number of threads should be actually higher than number of streaming threads. The task cancellation may take some time. Good to have some extra threads and not tie it to exactly number of streaming threads.

Reply:
could we comment this out and use enricherExecutorsThreadPoolSize if set, else 2*numStreamThreads ?